locale/en/LC_MESSAGES/django.po

```diff
@@ -2,7 +2,7 @@ msgid ""
 msgstr ""
```

Comment:
@Mitso can you re generate this file again with ignoring the `rules.py` as well, we don't need to add those since we don't translate the sentences that are shown in wagtail